packages/use-intl/test/react/useFormatter.test.tsx

```diff
@@ -209,6 +209,11 @@ describe('number', () => {
     screen.getByText('2,948,192,329.123');
   });
 
+  it('formats a bigint', () => {
+    renderNumber(294819232912312);
```

Comment:
This is a regular number, not a big int—right? See the comment above.

Reply:
Completely batched job. Apologies for that. Updated PR.